CanSat-code/CanSat-container/lib/XBEE.cpp

```diff
@@ -0,0 +1,37 @@
+#pragma once
+#include <SoftwareSerial.h>
+#include <Wire.h>
+
+#define RX_pin 12
+#define TX_pin 13
```

Comment:
Let's make a constructer for this class so that we can define what pins will be used for Rx and Tx